### PR TITLE
Document units of bus carriers

### PR DESCRIPTION
**Pull request by Irieo**
add explicit documentation of the unit of each bus' carrier
this addresses the issue https://github.com/PyPSA/pypsa-eur-sec/issues/134#event-6804897616

Bus [unit]:

- electricity, low voltage [MWh_el]
- home battery, battery, batteries in transport  [MWh_el]

- coal, nuclear and oil  [MWh_th]
- ice in transport [MWh_th]
- buses in heat sector [MWh_th]
- biogas and biomas [MWh_th]

- co2 atmosphere, co2 stored, process emissions [t_co2]

- H2, shipping_hydrogen_liquefaction [MWh_LHV]
- gas for industry [MWh_LHV]

### CODE REVIEW

File: scripts/prepare_sector_network.py [hunk 1]
```diff
@@ -385,10 +385,13 @@ def add_carrier_buses(n, carrier, nodes=None):
 
     n.add("Carrier", carrier)
 
+    unit="MWh_LHV" if carrier=="gas" else "MWH_th"
```
Comment (fneum) [inline scripts/prepare_sector_network.py:388]:
```suggestion
    unit = "MWh_LHV" if carrier == "gas" else "MWh_th"
```

File: scripts/prepare_sector_network.py [hunk 2]
```diff
@@ -2448,3 +2469,5 @@ def limit_individual_line_extension(n, maxext):
         add_electricity_grid_connection(n, costs)
 
     n.export_to_netcdf(snakemake.output[0])
+
+# %%
```
Comment (fneum) [inline scripts/prepare_sector_network.py:2473]:
```suggestion
```

**Review by fneum — CHANGES_REQUESTED** [review]
This is very handy!

Oil and biomass should also be labelled as LHV, I think.

**Comment by Irieo** [comment]
> This is very handy!
> Oil and biomass should also be labelled as LHV, I think.

@fneum done.

**Pull request merged**